core/src/Kokkos_CopyViews.hpp

```diff
@@ -862,14 +862,16 @@ template <class DstType, class SrcType, class ExecSpace>
 struct ViewRemap<DstType, SrcType, ExecSpace, 1> {
   using p_type = Kokkos::pair<int64_t, int64_t>;
 
-  ViewRemap(const DstType& dst, const SrcType& src) {
+  template <typename... OptExecSpace>
+  ViewRemap(const DstType& dst, const SrcType& src,
+            const OptExecSpace&... exec_space) {
```

Comment:
what is exec_space... ?? If it includes more then execspace the name should probably be different?

Reply:
It's just an execution space but it's optional to provide it.